corehq/apps/analytics/tasks.py

```diff
@@ -278,6 +279,8 @@ def track_periodic_data():
             if domain in domains_to_mobile_users and domains_to_mobile_users[domain] > max_workers:
                 max_workers = domains_to_mobile_users[domain]
 
+        project_spaces_created = get_domains_created_by_user(email)
+
```

Comment:
Why does this need periodic check, shouldn't it get triggered when a domain gets created?